router/batchrouter/asyncdestinationmanager/bing-ads/util.go

```diff
@@ -303,12 +311,44 @@ func ReadPollResults(filePath string) [][]string {
 	// Read all records from the CSV file
 	records, err := reader.ReadAll()
 	if err != nil {
-		log.Fatal("Error reading CSV:", err)
+		b.logger.Errorf("Error reading CSV: %w", err)
```

Comment:
don't we need to propagate error to the caller?

Reply:
done